crates/trigger/src/stdio.rs

```diff
@@ -15,16 +11,13 @@ pub enum FollowComponents {
     None,
     /// Only the specified components should have their logs followed.
     Named(HashSet<String>),
-    /// All components should have their logs followed.
-    All,
 }
 
 impl FollowComponents {
     /// Whether a given component should have its logs followed on stdout/stderr.
     pub fn should_follow(&self, component_id: &str) -> bool {
         match self {
-            Self::None => false,
-            Self::All => true,
+            Self::None => true,
```

Comment:
I should maybe wait for the refactoring but it seems like this gets rid of "all" and changes the semantics of "none" to "all" (and implements "none" via an empty list?).  If it results in following all components then call it "all"... *grin*